src/vttLib/__main__.py

```diff
@@ -1,103 +1,91 @@
-from __future__ import absolute_import, division, print_function
-
 import logging
+import os
 import sys
-from argparse import ArgumentParser, _VersionAction
-
-from vttLib import VTTLibArgumentError, vtt_compile, vtt_dump, vtt_merge
+from argparse import ArgumentParser
 
+import fontTools.ttLib
 
-class PkgVersionAction(_VersionAction):
-    def __call__(self, parser, namespace, values, option_string=None):
-        import pkg_resources
-
-        self.version = pkg_resources.get_distribution("vttLib").version
-        super(PkgVersionAction, self).__call__(parser, namespace, values, option_string)
+import vttLib
 
 
 def main(args=None):
     parser = ArgumentParser(
         prog="python -m vttLib",
-        description="Dump, merge or compile Visual "
-        "TrueType data in UFO3 with FontTools",
+        description="Dump, merge or compile Visual TrueType data with FontTools",
     )
-    parser.add_argument("--version", action=PkgVersionAction)
-
+    parser.add_argument("--version", action="version", version=vttLib.__version__)
     parser_group = parser.add_subparsers(title="sub-commands")
-
-    parser_dump = parser_group.add_parser(
-        "dump",
-        help="export VTT tables and 'maxp' values from TTF to UFO",
-        description="Dump VTT tables as TTX files to UFO3 'data' folder.",
+    parser_dumpfile = parser_group.add_parser(
+        "dumpfile",
+        description="Export VTT tables and 'maxp' values from a TTF into a TTX dump",
     )
-    parser_merge = parser_group.add_parser(
-        "merge",
-        help="import VTT source data stored in UFO to TTF",
-        description="Merge VTT tables from UFO3 'data' folder to TTF font.",
+    parser_mergefile = parser_group.add_parser(
+        "mergefile",
+        description="Write VTT source data stored in a TTX dump into a TTF.",
     )
     parser_compile = parser_group.add_parser(
         "compile",
-        help="generate TrueType bytecode from VTT assembly",
-        description="Generate 'fpgm', 'prep', 'cvt ' and 'glyf' programs from "
-        "VTT assembly.",
+        description=(
+            "Generate 'fpgm', 'prep', 'cvt ' and 'glyf' programs from VTT assembly."
+        ),
     )
-    subparsers = (parser_compile, parser_dump, parser_merge)
-
-    for subparser in subparsers:
+    parser_dumpfile_from_ufo = parser_group.add_parser(
+        "dumpfile_from_ufo", description="Dump VTT data from UFO3 data to a TTX dump."
```

Comment:
Please say here that the original data files will be removed, or don't remove the original data files (I would prefer to not remove the input files because it's unexpected for a tool that reads data to also delete it)